core/tests/puppeteer-acceptance-tests/blog-post-draft-blogs.js

```diff
@@ -0,0 +1,32 @@
+const acceptanceTests = require("./utility-functions/puppeteer_utils.js");
+const testConstants = require("./utility-functions/testConstants.js");
+
+
+const blogEditOptions = "e2e-test-blog-post-edit-box";
+
+async function deleteDraftAsBlogAdmin() {
+  const user = await new acceptanceTests();
+  const page = await user.init();
+
+  await user.goto(testConstants.URLs.home);
+  await user.clickOn("button", "OK");
+  await user.clickOn("span", "Sign in");
+  await user.type(testConstants.SignInDetails.inputField, "testadmin@example.com");
+  await user.clickOn("span", "Sign In");
+  
+  await user.goto(testConstants.URLs.BlogDashboard, testConstants.Dashboard.MainDashboard);
+
+  try{
+    await user.clickOn("button", blogEditOptions); // an icon
+    await user.clickOn("span", "Delete", 100);
+    await user.clickOn("button", " Confirm ");
+  } catch {
+    console.log("no blog post in drafts");
```

Comment:
Instead of doing this can you assert that there's at least one blog post first, and drop the try/catch?

Reply:
we can do this, but by this means, we are holding a condition that there must be atleast one draft blog in the blog dashboard wherever the tests are running, otherwise the test will fail and will be giving some errors in the log